History.markdown

```diff
@@ -1,3 +1,5 @@
+# History
+
```

Comment:
I would prefer to leave this out. I believe our changelog generator will nix it anyway.